docs/learn/guides/file-integrations/file-integration-using-ftp/README.md

```diff
@@ -0,0 +1,198 @@
+# File Integration using FTP
+
+The WSO2 FTP Connector lets you connect to an FTP server, and perform operations on files and folders stored on the 
+server. These operations include basic file operations such as reading, updating and deleting files, and listening to 
+the server to invoke operations when a file is created or deleted.
+
+> In this guide you will learn about how to use the WSO2 FTP Connector to create an FTP listener service using Ballerina.
+
+The following are the sections available in this guide.
+
+- [What you'll build](#what-youll-build)
+- [Prerequisites](#prerequisites)
+- [Implementation](#implementation)
+- [Testing](#testing)
+- [Deployment](#deployment)
+
+## What you'll Build
+
+To understand how to build a service to listen to an FTP server, let's consider the use case of a data center which uses 
+an FTP server to store data files. When a new file is added to the server, the FTP listener will read the file and add 
+the file name and size to a map, and when the file is deleted from the server, it will remove the entry from the map. 
+
+## Prerequisites
+ 
+- [Ballerina Distribution](https://ballerina.io/learn/getting-started/)
+- A Text Editor or an IDE 
```

Comment:
We have to change this to only use VS Code. This is the recommended IDE for Ballerina Integrator.

Reply:
Fixed.